back-tracking/Sudoku.js

```diff
@@ -0,0 +1,86 @@
+function findEmptyCell (board) {
```

Comment:
as we are passing board everywhere, what about creating classes. we use ES6 which introduced classes.
it will be a good idea to use classes
```

class Sudoku {
  constructor(board) {
    this.board = board
  }
}
```